packages/td-tools/src/thing-model-helpers.ts

```diff
@@ -241,7 +241,7 @@ export class ThingModelHelpers {
             case "http": {
                 return new Promise((resolve, reject) => {
                     http.get(uri, (res) => {
-                        if (!res.statusCode || res.statusCode !== 200) {
+                        if (res.statusCode === undefined || res.statusCode !== 200) {
```

Comment:
```suggestion
                        if (res?.statusCode !== 200) {
```

Reply:
I don't think the proposed change leads to the same result ---> reverting for now..